CMakeLists.txt

```diff
@@ -345,6 +346,22 @@ else (HAVE_USING_OPERATOR)
   set (ac_cv_cxx_using_operator 0)
 endif (HAVE_USING_OPERATOR)
 
+if (HAVE_EXECINFO_H)
+  set (HAVE_STACKTRACE 1)
+endif (HAVE_EXECINFO_H)
+
+if (WIN32)
+  set (HAVE_STACKTRACE 1)
```

Comment:
The Windows stacktrace support also comes from the `DbgHelp` library, maybe this should be moved under the `if (HAVE_DBGHELP)`.